crates/sui-indexer/migrations_v2/2023-08-19-044052_epochs/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE epochs
```

Comment:
There's quite a few things in `type Epoch` from the GraphQL schema that aren't here.  Maybe they are in some different place? Let me know, because I think they are mostly to do with the system state, but I couldn't find another home for it in the new schemas:

- system state version
- protocol config
- system parameters
- stake subsidy (balance, distribution counter, current distribution amount, period length, decrease rate).
- validator set (total stake, active validators, etc).
- storage fund (storage rebates, non refundable balance).
- safe mode (enabled, gas cost summary.  This one is actually to do with epochs, and not the system in general)
- end timestamp (ditto -- but maybe this is in end of epoch data?)

Also what's the difference between `end_of_epoch_info` and `end_of_epoch_data`?

Reply:
as discussed, let's move this offline